package.json

```diff
@@ -7,24 +7,40 @@
   },
   "description": "Convert html to rtf format in the server",
   "readmeFilename": "README.md",
-  "keywords": ["html", "rtf", "convert", "server", "node", "javascript"],
+  "keywords": [
+    "html",
+    "rtf",
+    "convert",
+    "server",
+    "node",
+    "javascript"
+  ],
   "license": "MIT",
   "main": "index.js",
+  "module": "dist/html-to-rtf.js",
+  "exports": {
+    ".": {
+      "require": "./dist/html-to-rtf.umd.cjs",
+      "import": "./dist/html-to-rtf.js"
```

Comment:
Consume the ESM or CJS version based on whether you `import` or `require` this library.